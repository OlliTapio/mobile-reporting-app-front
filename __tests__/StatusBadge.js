import 'react-native';
import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import StatusBadge from '../src/components/StatusBadge/StatusBadge';

configure({ adapter: new Adapter() });

it('renders correctly', () => {
    const tree = renderer.create(
        <StatusBadge dateCreated='2018-04-01' />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});





