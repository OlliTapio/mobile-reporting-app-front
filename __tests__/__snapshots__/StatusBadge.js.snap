// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={undefined}
>
  <View
    style={
      Array [
        undefined,
      ]
    }
  >
    <View
      onPress={undefined}
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#43484d",
            "borderRadius": 20,
            "justifyContent": "center",
            "padding": 12,
            "paddingBottom": 3,
            "paddingTop": 3,
          },
          Array [
            undefined,
            Object {
              "backgroundColor": "#99d9ad",
            },
          ],
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={undefined}
      >
        Approved
      </Text>
      <View
        onPress={undefined}
        style={
          Array [
            false,
            false,
            false,
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            },
            undefined,
          ]
        }
        underlayColor="white"
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 24,
              },
              Array [
                Object {
                  "backgroundColor": "transparent",
                },
                undefined,
              ],
              Object {
                "fontFamily": "Feather",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          юде
        </Text>
      </View>
    </View>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={undefined}
  />
</View>
`;
